### PR TITLE
Guide: Delete all stay signed in sessions

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adds a small guide that describes how to "delete all sessions"
- **Is this PR related to a Monolith release?** Yes. 2024.02.0
### Resolves:

* [OKTA-649888](https://oktainc.atlassian.net/browse/OKTA-649888)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/index.md](https://github.com/okta/okta-developer-docs/blob/0bfd00a19f210052f8cc8f08ace6702b30ff5142/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdelete-all-stay-signed-in-sessions%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7ecf700c-e6e5-41f0-941d-819461e5f778) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md](https://github.com/okta/okta-developer-docs/blob/0bfd00a19f210052f8cc8f08ace6702b30ff5142/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdelete-all-stay-signed-in-sessions%2Fmain%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e5009260-e4d9-4afb-ba5e-7e59dbd81b21) | 74  | 85  | 74  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/0bfd00a19f210052f8cc8f08ace6702b30ff5142/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f0153772-c20b-4084-a8d7-5b94a3317262) | 55  | 55  | 86  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.889fbaa9-5971-42c1-b2d8-617509144b49?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+---
+title: Delete all Stay signed in sessions
+---
+
+<ApiLifecycle access="ie" />
+
+This guide explains how to delete all sessions of the **Stay signed in** feature using the [My Account Sessions API](https://developer.okta.com/docs/api/openapi/okta-myaccount/myaccount/tag/Sessions/). See [Stay signed in](https://help.okta.com/okta_help.htm?type=oie&id=ext-stay-signed-in) for more details.
+
+> **Note:** This document is written for Okta Identity Engine. If you’re using Classic Engine, time to upgrade. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md:9]:
```suggestion
> **Note:** This document is written for Okta Identity Engine. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
```

Comment (grahamsmith-okta) [reply packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md:9]: I really like the conversational "time to upgrade" line but I'm not sure it'd work for translation purposes.

File: packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+**Sample code**
+
+[Request example](#request-example)
+
+---
+
+## About **Stay signed in**
+
+**Stay signed in** lets users establish an Okta session that continues across browsers and devices. When a user selects this option during authentication, they won't be prompted again for MFA for the amount of time defined in your global session policy.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md:32]:
```suggestion
**Stay signed in** lets users establish an Okta session that continues after they close and reopen their browsers. When a user selects this option during authentication, they won't be prompted again for MFA for the amount of time defined in your global session policy.
```

Comment (grahamsmith-okta) [reply packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md:32]: I had to change my opening line in the product docs for this reason, stressing that it didn't cross devices necessarily

File: packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+Persistent SSO: Persistent SSO cookies are written for the authenticated user, which eliminates further prompts when the user switches applications for as long as the persistent SSO cookie is valid.
+
+The difference between persistent SSO and session SSO is that you can maintain persistent SSO across different sessions. Persistent SSO is disabled by default in Okta. To share a sign-in session across different sessions, you need to enable persistent SSO.
+
+### Enable the parameter
+
+To enable persistent SSO for **Stay signed in**, you need to update the [`usePersistentCookie`](/docs/reference/api/policy/#signon-action-object) parameter in the appropriate global session policy. These steps use the Okta [Policy Postman collection](/docs/reference/postman-collections/) to enable the parameter.
+
+Before you begin, be sure to:
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md:59]:
```suggestion
Before you begin, complete the following steps:
```

Comment (grahamsmith-okta) [reply packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md:59]: Globalization team requests that all stems function as complete sentences (they don't read the list items as complementary to the stem fragment). This is just something we've started doing on prod side. Will be required when dev docs start getting translated.

**Review by grahamsmith-okta — APPROVED** [review]
A couple of small suggestions, otherwise good to go!

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/index.md](https://github.com/okta/okta-developer-docs/blob/3560831a55905430dac539f808ccfc11b3372ce3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdelete-all-stay-signed-in-sessions%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/eed688f3-c042-4eec-bb04-1022b339b172) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md](https://github.com/okta/okta-developer-docs/blob/3560831a55905430dac539f808ccfc11b3372ce3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdelete-all-stay-signed-in-sessions%2Fmain%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/95477873-3359-468e-83b3-27e3af2491b7) | 74  | 85  | 74  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/3560831a55905430dac539f808ccfc11b3372ce3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/dcf39c25-d39e-42e9-922c-5cd56de4f463) | 55  | 55  | 86  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.8e7a399d-792a-472a-9e21-cf394fe25c01?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/index.md](https://github.com/okta/okta-developer-docs/blob/cbdce8d87b56b51fbef97ffcab0d806dc7d1b280/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdelete-all-stay-signed-in-sessions%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ff49be85-3d01-4415-8204-384d97aca2c8) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md](https://github.com/okta/okta-developer-docs/blob/cbdce8d87b56b51fbef97ffcab0d806dc7d1b280/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdelete-all-stay-signed-in-sessions%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/216dc5cb-3d1d-459f-b353-f98505d8f61e) | 74  | 85  | 74  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/cbdce8d87b56b51fbef97ffcab0d806dc7d1b280/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/42de5c92-9aa5-42d9-a182-662b869f11b5) | 55  | 55  | 86  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.6c3ff051-ec0e-4fa0-8a15-09e5444c2710?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — APPROVED** [review]
Looks good!

**Pull request merged**